cumulusci/tasks/bulkdata/generate_mapping.py

```diff
@@ -291,6 +291,49 @@ def _split_dependencies(self, objs, dependencies):
 
         return stack
 
+    def find_free_object(self, objs_remaining: list, dependencies: dict):
+        # if you change this code, remember that
+        # peeking into a generator consumes it
+        free_objs = (
+            sobj
+            for sobj in objs_remaining
+            if only_has_soft_dependencies(sobj, dependencies[sobj])
+        )
+        first_free_obj = next(free_objs, None)
+        # first_free_obj = (
+        #     "Account" if "Account" in objs_remaining else next(free_objs, None)
+        # )
```

Comment:
What's the purpose of this comment? A reminder of an alternative we can try if we need to?

Reply:
Removed.